src/components/application_manager/src/resumption/resumption_data_processor_impl.cc

```diff
@@ -307,7 +313,7 @@ bool ResumptionDataProcessorImpl::IsResumptionSuccessful(
   sync_primitives::AutoReadLock lock(resumption_status_lock_);
   auto it = resumption_status_.find(app_id);
   if (resumption_status_.end() == it) {
-    return false;
+    return true;
```

Comment:
@ychernysheva why is it successful in that case?

Reply:
@AKalinich-Luxoft because we need consider successful cases, when app is not subscribed to any HMI responses (in our situation such apps include apps without any HMI subscriptions, but with subscriptions to AppServices, for example, and also apps, which have subscriptions to waypoints, vd or ivd, which doesn't transfer to DataProcessor with fake responses)